llama-index-integrations/vector_stores/llama-index-vector-stores-milvus/llama_index/vector_stores/milvus/base.py

```diff
@@ -50,15 +50,16 @@ def extract_host_port(url):
     return host, port
 
 
-def _to_milvus_filter(standard_filters: MetadataFilters) -> List[str]:
+def _to_milvus_filter(standard_filters: MetadataFilters) -> str:
     """Translate standard metadata filters to Milvus specific spec."""
     filters = []
     for filter in standard_filters.legacy_filters():
```

Comment:
for future note, using `legacy_filters()` here means it will only support exact match filters (even though there are many many filter types supported in the enums)